monai/apps/pathology/engines/utils.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional, Sequence, Union
+
+import torch
+
+from monai.engines import PrepareBatch, PrepareBatchExtraInput
+from monai.utils import ensure_tuple
+from monai.utils.enums import HoVerNetBranch
+
+__all__ = ["PrepareBatchWithDictLabel"]
+
+
+class PrepareBatchWithDictLabel(PrepareBatch):
```

Comment:
It seems to be very specific to hovernet, so might be better to change the name to `PrepareBatchHoVerNet`. Thanks

Reply:
I agree, actually I also raised this concern to @KumoLiu offline.

Thanks.

Reply:
Hi @drbeh @Nic-Ma, thanks for your suggestion, I will update it in the latest commit.